### PR TITLE
#679 Header Search icons place Close far-right same as Search fix

### PR DESCRIPTION
**Pull request by amolv**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -919,7 +919,7 @@ section {
 .section-inner {
 	margin-left: auto;
 	margin-right: auto;
-	max-width: 120rem;
```
Comment (andersnoren) [inline style.css]: This helper class is used for other elements than the search modal, so you can change this line back to `120rem`.

Comment (amolv) [reply style.css]: Sorry, its back to 120rem now.

**Comment by andersnoren** [comment]
@amolv What's your WordPress.org username?

**Comment by amolv** [comment]
> @amolv What's your WordPress.org username?

amolv - https://profiles.wordpress.org/amolv/

**Pull request merged**